### DISCUSSION TITLE
Ruby 3.1.0 support?

### DISCUSSION THREAD

**Discussion by MGatner** [rbenv/ruby-build#1925] [Build failures] [answered]
Sorry if I am missing something obvious, I'm new to `rbenv`. I was running dandy on Ruby 3.0.2 when a new project requires 3.1.0. "No problem," I thought, "I use `rbenv`!" I pull the latest and try to install, but "Definition not found: 3.1.0". Sure enough `rbenv install -L`:
```bash
...
2.6.7
2.6.8
2.7.0-dev
2.7.0-preview1
2.7.0-preview2
2.7.0-preview3
2.7.0-rc1
2.7.0-rc2
2.7.0
2.7.1
2.7.2
2.7.3
2.7.4
3.0.0-dev
3.0.0-preview1
3.0.0-preview2
3.0.0-rc1
3.0.0
3.0.1
3.0.2
3.1.0-dev
jruby-1.5.6
jruby-1.6.3
...
```

Is this something still in the works, or did I mess up my environment? 

OS: Ubuntu 20.04
`rbenv`: Latest `dev-master`

**Comment by pedrosfdcarneiro** [comment, marked as answer]
Have you tried to update the `ruby-build` plugin? It seems to me that may be the problem here.

If you have installed `ruby-build` as a plugin, it's simple as running the following shell command:
`git -C "$(rbenv root)"/plugins/ruby-build pull` (and then run the `rbenv install` command).

See https://github.com/rbenv/ruby-build#upgrading to know more about it.

**Comment by MGatner** [comment]
Thanks, I will try this and post back.

**Comment by MGatner** [comment]
Yep! That was it. Sorry for the mistake. As feedback from a noob-ish user, it wasn't apparent to me that this was different than pulling the latest `master` branch from `rbenv` root.
